dz5/home/admin.py

```diff
@@ -9,7 +9,12 @@
 
 class StudentAdmin(ModelAdmin):
 
-    list_display = ("name", "email", "birthday")
+    list_display = ("name", "email", "birthday", "students_social")
+
+    def students_social(self, object):
+        social = object.url
```

Comment:
If student has social url you should return name and surname like a link, if student doesn't have return just name and surname